swagger.yaml

```diff
@@ -218,34 +236,31 @@ paths:
           $ref: '#/responses/InternalError'
     put:
       tags:
-        - Public
-        - Private
-      summary: Update a filter
-      description: >-
-        Update the filter by providing new properties, submit a filter for
-        processing by setting query parameter `submitted` to `true`
+      - "Public"
+      summary: "Update a filter"
+      description: "Update the filter by providing new properties, submit a filter for processing by setting query parameter `submitted` to `true`"
```

Comment:
This process isn't true for us. This was a hack in the old api Eleanor asked my to change if I could (I didn't in the existing one but no point copying it in ours), we're going to have the separate /query endpoint to trigger ours. I don't think we need any of those URL parameters. No idea what that if-match parameter is all about it's actually a header.

Reply:
This has now been removed.